usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -61,19 +65,29 @@ def handle(self, *args, **options):
                 self.initial_run()
                 return
 
-            self.logger.info(f"Running delete SQL for '{self.etl_level}' ETL")
-            self.spark.sql(self.delete_records_sql())
-
             # Capture start of the ETL to update the "last_load_date" after completion
             etl_start = datetime.now(timezone.utc)
 
-            last_etl_date = str(get_last_load_date(self.etl_level, lookback_minutes=15))
+            self.logger.info(f"Running delete SQL for '{self.etl_level}' ETL")
+            self.spark.sql(self.delete_records_sql())
+
+            last_etl_date = get_last_load_date(self.etl_level, lookback_minutes=15)
+            if last_etl_date is None:
+                # Table has not been loaded yet.  To avoid checking for None in all the locations where
+                # last_etl_date is used, set it to a long time ago.
+                last_etl_date = datetime.utcfromtimestamp(0)
+            self.last_etl_date = str(last_etl_date)
 
             self.logger.info(f"Running UPSERT SQL for '{self.etl_level}' ETL")
             if self.etl_level == "transaction_id_lookup":
-                self.load_transaction_lookup_ids(last_etl_date)
+                self.update_transaction_lookup_ids()
             elif self.etl_level == "award_id_lookup":
-                self.load_award_lookup_ids(last_etl_date)
+                self.update_award_lookup_ids()
+            else:
```

Comment:
Why is there an extra `else` here? Can't this condition chain just continue with 
```
elif self.etl_level == "transaction_fabs":
```

Reply:
There had been some more code in that block after the else, but I consolidated it down to just one function call each, so you're right, there is no need for that else any more.